src/components/Paginate.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react'
 import { RingLoader } from 'react-spinners'
 
-const Paginate = ({ items, Component, error }) => {
+const Paginate = ({ items, Component, error, fetchUserInfo }) => {
```

Comment:
Why is `fetchUserInfo` part of the `Paginate` component?
This looks specific to `UserProfile`

Reply:
sorry, not sure why that was there, good catch